datasets/xk/companies/crawler.py

```diff
@@ -0,0 +1,291 @@
+    """
+    Fetch a single company from the Kosovo Registry of Business Organizations and Trade Names.
+    """
+    try:
+        resp = context.fetch_json(
+            "https://arbk.rks-gov.net/api/api/Services/TeDhenatBiznesit",
+            params={"nRegjistriId": company_id, "Gjuha": "en"},
+            headers={"key": get_the_key()},
+        )
+
+        if len(resp) != 1 or "teDhenatBiznesit" not in resp[0]:
+            context.log.error(f"Something strange with the {company_id})")
+
+        company = resp[0]["teDhenatBiznesit"]
+        activities = resp[0].get("aktivitetet", [])
+        founders = resp[0].get("pronaret", [])
+        representatives = resp[0].get("perfaqesuesit", [])
+
+        company_type = company.get("LlojiBiznesit")
+        if company_type not in COMPANY_TYPES:
+            context.log.warning("Unknown company type: ", company_type)
+
+        status = company.get("StatusiARBK")
+        if status not in STATUSES:
+            context.log.warning("Unknown status: ", status)
+
+        entity = context.make("Company")
+        entity.id = context.make_id("XKCompany", company_id)
+        entity.add("country", "xk")
+
+        for orig_field, field_def in FIELDS_MAPPING.items():
+            if company.get(orig_field):
+                entity.add(
+                    field_def["field"],
+                    company.pop(orig_field),
+                    lang=field_def.get("lang"),
+                )
+
+        if company_type in COMPANY_TYPES:
+            entity.add("legalForm", COMPANY_TYPES[company_type], lang="eng")
+
+        if status in STATUSES:
+            entity.add("status", STATUSES[status], lang="eng")
+
+        if company.get("DataRegjistrimit"):
+            entity.add("incorporationDate", parse_date(company.pop("DataRegjistrimit")))
+
+        if company.get("DataShuarjesBiznesit"):
+            entity.add(
+                "dissolutionDate", parse_date(company.pop("DataShuarjesBiznesit"))
+            )
+
+        if company.get("Kapitali"):
+            capital = norm_capital(context, company.pop("Kapitali"))
+            if capital["value"]:
+                entity.add("capital", capital["value"])
+
+            if capital["currency"]:
+                entity.add("currency", capital["currency"])
+
+        if company.get("Adresa") or company.get("Vendi") or company.get("Komuna"):
+            address = h.make_address(
+                context,
+                street=company.pop("Adresa"),
+                city=company.pop("Vendi"),
+                country_code="xk",
+                # Should it be county or region?
+                state=company.pop("Komuna"),
+            )
+            h.apply_address(context, entity, address)
+
+        if company.get("NumriPunetoreve"):
+            entity.add(
+                "notes",
+                f'Number of employees: {company.pop("NumriPunetoreve")}',
+                lang="eng",
+            )
```

Comment:
I think you can drop this for now - it's already going to be a LOT of data and I don't think this is a priority.